addons/survey/models/survey_question.py

```diff
@@ -513,7 +513,11 @@ def _prepare_statistics(self, user_input_lines):
         """ Compute statistical data for questions by counting number of vote per choice on basis of filter """
         all_questions_data = []
         for question in self:
-            question_data = {'question': question, 'is_page': question.is_page}
+            has_image = False
```

Comment:
Could we keep that in the template to avoid bloating the controller returned data?

Would be something like

```xml
<t t-set="question_has_image_answers"
    t-value="any(image for image in question.suggested_answer_ids.value_image_filename)"/>
```

Reply:
changed it accordingly, now it's calculated in the template instead :smile: